lib/widgets/theme.dart

```diff
@@ -75,19 +75,33 @@ const kZulipBrandColor = Color.fromRGBO(0x64, 0x92, 0xfe, 1);
 /// For how to export these from the Figma, see:
 ///   https://www.figma.com/design/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=2945-49492&t=MEb4vtp7S26nntxm-0
 class DesignVariables extends ThemeExtension<DesignVariables> {
-  DesignVariables() :
-    bgMain = const Color(0xfff0f0f0),
-    bgTopBar = const Color(0xfff5f5f5),
-    borderBar = const Color(0x33000000),
-    icon = const Color(0xff666699),
-    title = const Color(0xff1a1a1a),
-    streamColorSwatches = StreamColorSwatches.light;
+  factory DesignVariables.light() {
+    return DesignVariables._(
```

Comment:
nit: can simplify slightly:
```suggestion
  DesignVariables.light() :
    this._(
```

That's a Dart "redirecting constructor":
https://dart.dev/language/constructors#redirecting-constructors